test/unit/org/apache/cassandra/gms/GossiperTest.java

```diff
@@ -190,6 +191,7 @@ public void testLargeGenerationJump() throws UnknownHostException, InterruptedEx
         {
             // clean up the gossip states
             Gossiper.instance.endpointStateMap.clear();
+            Gossiper.instance.gossipStateTransitionMap.clear();
```

Comment:
to be more maintainable should we not have a `cleanup` method?  Seeing a lot of other maps not cleaned up, so this gets corrupted

Reply:
sounds good